client/src/gamedata/cn_simplified/descriptions/levels/instances.md

```diff
@@ -38,32 +38,32 @@ help()
 ethernaut
 ```
 
-这个游戏的主要合约，你不需要通过控制台和他直接互动（因为这个应用已经替你做好了），但是如果你想，你还是可以做到。先尝试熟悉它，这是让你了解如何和游戏里其他合约互动的好途径。
+这个游戏的主要合约，你不需要通过控制台和他直接互动（因为这个应用已经替你做好了），但是如果你想，你还是可以做到。先尝试熟悉它，这是让你了解如何和游戏里其他合约互动的好方法。
 
 然后让我们来展开 ethernaut 看看里面都有啥。
 
 &nbsp;
 #### 5. 和 ABI 互动
-`ethernaut` 是一个 `TruffleContract` 对象， 它包装了部署在区块链上的 `Ethernaut.sol` 合约contract that has been deployed to the blockchain.
+`ethernaut` 是一个 `TruffleContract` 对象， 它包装了部署在区块链上的 `Ethernaut.sol` 合约.
 
-除此之外，合约的 ABI 还暴露所有的 `Ethernaut.sol` 公开方法, 比如 `owner`. 比如输入以下命令:
+除此之外，合约的 ABI 还提供了所有的 `Ethernaut.sol` 公开方法, 比如 `owner`. 比如输入以下命令:
 ```
 ethernaut.owner()
 ```
-###### `await ethernaut.owner()` 如果你使用的是 Chrome v62.
-你可以看到这个 ethernaut 所有者是谁, 当然不是你，哈哈 XD.
+######  如果你使用的是 Chrome v62, 可以使用 `await ethernaut.owner()`.
+你可以看到这个 ethernaut 合约的所有者是谁, 当然不是你，哈哈 XD.
 
 &nbsp;
 #### 6. 获得测试ether
-你需要ether来进行这个游戏. 在测试网络上获得ether最简单的方法是通过 [this](https://faucet.rinkeby.io/), [this](https://faucets.chain.link/rinkeby) 或 [this faucet](https://faucet.paradigm.xyz/).
+你需要ether来进行这个游戏. 在测试网络上获得ether最简单的方法是通过 [this faucet](https://faucet.rinkeby.io/).
```

Comment:
Can we leave the three faucets instead of just one ? faucet.rinkeby.io has been unreliable lately.